src/js/events/MesosSummaryActions.js

```diff
@@ -104,12 +104,17 @@ var MesosSummaryActions = {
     function (resolve, reject) {
 
       return function (timeScale) {
-        let canAccessHistoryServer = Hooks.applyFilter(
+        let canAccessHistoryAPI = Hooks.applyFilter(
           'hasCapability', false, 'historyServiceAPI'
         );
 
-        if (!_historyServiceOnline || !canAccessHistoryServer) {
-          requestFromMesos(resolve, reject);
+        if (!_historyServiceOnline || !canAccessHistoryAPI) {
+          let canAccessMesosAPI = Hooks.applyFilter(
+            'hasCapability', false, 'mesosAPI'
+          );
+          if (canAccessMesosAPI) {
```

Comment:
Wouldn't this mean that we will never make the request in some cases, which would in turn never load the UI?